### ISSUE TITLE
After closing and opening the lid with purge-nvda screen stays black 

### ISSUE THREAD

**Issue by PwnedBOT** [mayankk2308/purge-nvda#14]
When I install purge-nvda and close and reopen the lid, the screen stays black does not light up again until I reboot.

**Comment by mayankk2308** [comment]
This is a known issue - I cannot develop this script further for more workarounds as I do not have an applicable Mac. I believe someone made a kext so solve the sleep problem. Will find the link and share here once I'm free for the day.

**Comment by PwnedBOT** [comment]
Ok nice

**Comment by mayankk2308** [comment]
Hi @PwnedBOT, this was the [article](https://tim.palpant.us/sysadmin/macos/2018/09/09/nvda-gpu-wake-handler/) I had come across.

**Comment by PwnedBOT** [comment]
@mayankk2308 Is it the "Enable backlight after waking from sleep in High Sierra (10.13) / Mojave (10.14)"-Step?

**Comment by mayankk2308** [comment]
@PwnedBOT I haven't read through it deeply, and it was a while back so I'm not sure, but probably. Contact the developer for more information on that.
